src/vdm.rs

```diff
@@ -0,0 +1,26 @@
+pub const PD_VDM_DATA_OBJ_SIZE: usize = 4;
+pub const PD_VDM_MAX_VDOS: usize = 6;
+pub const PD_VDM_MAX_NUM_DATA_OBJECTS: usize = 7;
+pub const PD_VDM_MAX_SIZE: usize = PD_VDM_DATA_OBJ_SIZE * PD_VDM_MAX_NUM_DATA_OBJECTS;
+pub const PD_VDM_OBJ_POS_ALL_MODES: u8 = 7;
+pub const PD_VDM_HEADER_VERSION: u8 = 1;
+
+/// SVDM header commands
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+pub enum VdmCmd {
```

Comment:
You're already in the `embedded_usb_pd::vdm` namespace so namespacing the items is redundant
```suggestion
pub const DATA_OBJ_SIZE: usize = 4;
pub const MAX_VDOS: usize = 6;
pub const MAX_NUM_DATA_OBJECTS: usize = 7;
pub const MAX_SIZE: usize = DATA_OBJ_SIZE * MAX_NUM_DATA_OBJECTS;
pub const OBJ_POS_ALL_MODES: u8 = 7;
pub const HEADER_VERSION: u8 = 1;

/// SVDM header commands
#[derive(Copy, Clone, Debug, PartialEq, Eq)]
#[cfg_attr(feature = "defmt", derive(defmt::Format))]
pub enum Cmd {
```